### PR TITLE
feat: bedrock model runtime enhancement

### LINKED ISSUE #6298: Modify most of bedrock model invoke method to converse api
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Modify most of bedrock model invoke method to converse api.

### 2. Additional context or comments

_No response_

### 3. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

### PR DESCRIPTION
**Pull request by longzhihun**
…d model invoke method to converse api

# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [x] https://github.com/langgenius/dify/issues/6298
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description
1. modify non stream tool use 
2. modify most of bedrock models invoking method to conversational api

Close https://github.com/langgenius/dify/issues/6298
Fix #6299

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: api/core/model_runtime/model_providers/bedrock/llm/llm.py [hunk 1]
```diff
@@ -48,6 +48,28 @@
 
 class BedrockLargeLanguageModel(LargeLanguageModel):
 
+    # please refer to the documentation: https://docs.aws.amazon.com/bedrock/latest/userguide/conversation-inference.html
+    # TODO There is invoke issue: context limit on Cohere Model, will add them after fixed.
+    CONVERSE_API_ENABLED_MODEL_INFO=[
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/bedrock/llm/llm.py:53]: I think we need to move these new features `support_system_prompts` & `support_system_prompts` to yaml files. Thoughts? @laipz8200 @takatost

**Review by crazywoola — COMMENTED** [review]
See comments

File: api/core/model_runtime/model_providers/bedrock/llm/llm.py (hunk 1 above)
Comment (longzhihun) [reply api/core/model_runtime/model_providers/bedrock/llm/llm.py:53]: Yes, i think so, but didn't know if there is a yaml file format standard?

Comment (longzhihun) [reply api/core/model_runtime/model_providers/bedrock/llm/llm.py:53]: > I think we need to move these new features `support_system_prompts` & `support_system_prompts` to yaml files. Thoughts? @laipz8200 @takatost

Just checked in model_entities.py, there is no key refer to system prompts, would you add it later? or what should I do next?

**Comment by laipz8200** [comment]
I think we can merge this PR first, then check how many models don't have a system prompt.

**Review by crazywoola — APPROVED** [review]

**Pull request merged**